be/src/Project/Services/Identity/Identity.Api/Configurations/AppSettings.cs

```diff
@@ -4,6 +4,7 @@ namespace Identity.Api.Configurations
 {
     public class AppSettings
     {
+        public string ClientUrl { get; set; }
```

Comment:
**Question** This property is in deployment files?

Reply:
Yes, it is in docker-compose.override and has value "http://client-api"